superbench/benchmarks/micro_benchmarks/gemm_flops_performance.py

```diff
@@ -24,16 +24,25 @@ def __init__(self, name, parameters=''):
 
         self._bin_name = 'cutlass_profiler'
 
+        # TODO - To support more architecutres, currently only support compute capability = 7.x or 8.x
```

Comment:
only support 7.0 and 8.0? 7.0 and 7.5 are different

Reply:
e.g., Volta refers to sm7.0, Turing refers to sm7.5, they have very different tensor cores.

Reply:
Do you know how to get the exact kernel name for different arch? Can we say that same arch have same kernel name?
For example, both 8.0 and 8.6 are ampere, do they have same gemm kernels?

Reply:
nope, not sure about that, but 7.5 introduces int tensor core which 7.0 does not have.
for sm8.0 and sm8.6, maybe need to compile cutlass profiler on both arch then check generated kernels?